src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -81466,6 +81469,527 @@ public U Test<U>(U val)
             var comp = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
             comp.VerifyDiagnostics();
         }
+
+        [Fact]
+        public void AttributeArgument_Constructor_NullLiteral()
```

Comment:
Consider putting these with the rest of the attribute tests, one of which seems to have been adjusted above.